src/main/java/com/google/code/twig/standard/StandardObjectDatastore.java

```diff
@@ -97,6 +97,20 @@ public TypeConverter getTypeConverter()
 		}
 		return converter;
 	}
+
+
+	public void resetTypeConverters() {
```

Comment:
that is the idea, to be called after instantiation, in a constructor, just to work around the static converters  = force registration again